dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes made to this file appear to be the addition of a new image asset named "Placeholder" with three different scale variants (1x, 2x, and 3x). This is a common practice for iOS development to ensure the image assets are properly scaled for different device resolutions.

The changes look correct and appropriate for adding a new image asset to the project.